src/web/styles.scss

```diff
@@ -136,3 +136,32 @@ body {
 .editor.editor-disabled {
   cursor: not-allowed;
 }
+
+@keyframes myExpand {
+  from {
+    opacity: 0;
+    max-height: 0;
+  }
+
+  to {
+    opacity: 1;
+    max-height: 800px;
+  }
+}
+
+.collapse {
+  transition: .35s ease-in-out;
+  opacity: 0;
+  visibility: collapse;
+  max-height: 0;
+  display: block !important;
+  overflow: hidden;
+
+  &.show {
+    opacity: 1;
+    max-height: 99999999px;
```

Comment:
What's the difference between this and the `800px` defined above?

Reply:
Using `99999999px` for both will cause the animation to instantly expand since it is expecting a much larger height.

The `800px` is used as a middle ground for the animation to play. The animation will play as if the max-height is 800px, but the final max-height would be `99999999px` to prevent overflow issues.